root/components/list/EventList.js

```diff
@@ -52,115 +55,75 @@ const EventList = ({
-            {sortable
-              ? (
-                <SortableTableHeader
-                  label={l('Type')}
-                  name="type"
-                  order={order}
-                />
-              )
-              : l('Type')}
-          </th>
-        ) : null}
-        {showArtists ? <th>{l('Artists')}</th> : null}
-        {artistRoles ? <th>{l('Role')}</th> : null}
-        {showLocation ? <th>{l('Location')}</th> : null}
-        <th>
-          {sortable
-            ? (
-              <SortableTableHeader
-                label={l('Date')}
-                name="date"
-                order={order}
-              />
-            )
-            : l('Date')}
-        </th>
-        <th>{l('Time')}</th>
-        {showRatings ? <th>{l('Rating')}</th> : null}
-      </tr>
-    </thead>
-    <tbody>
-      {events.map((event, index) => (
-        <tr className={loopParity(index)} key={event.id}>
-          {$c.user_exists && checkboxes ? (
-            <td>
-              <input
-                name={checkboxes}
-                type="checkbox"
-                value={event.id}
-              />
-            </td>
-          ) : null}
-          {seriesItemNumbers ? (
-            <td style={{width: '1em'}}>
-              {seriesItemNumbers[event.id]}
-            </td>
-          ) : null}
-          <td>
-            <DescriptiveLink entity={event} />
-          </td>
-          {showType ? (
-            <td>
-              {event.typeName
-                ? lp_attributes(event.typeName, 'event_type')
-                : null}
-            </td>
-          ) : null}
-          {showArtists ? (
-            <td>
-              <ArtistRoles relations={event.performers} />
-            </td>
-          ) : null}
-          {artist && artistRoles ? (
-            <td>
-              {event.performers.map(performer => (
-                performer.entity.id === artist.id ? (
-                  commaOnlyList(localizeArtistRoles(performer.roles))
-                ) : null
-              ))}
-            </td>
-          ) : null}
-          {showLocation ? (
-            <td>
-              <EventLocations event={event} />
-            </td>
-          ) : null}
-          <td>{formatDatePeriod(event)}</td>
-          <td>{event.time}</td>
-          {showRatings ? (
-            <td>
-              <RatingStars entity={event} />
-            </td>
-          ) : null}
-        </tr>
-      ))}
-    </tbody>
-  </table>
-);
+}: Props) => {
+  const columns = React.useMemo(
+    () => {
+      const checkboxColumn = $c.user_exists && checkboxes
+        ? defineCheckboxColumn(checkboxes)
+        : null;
+      const seriesNumberColumn = seriesItemNumbers
+        ? defineSeriesNumberColumn(seriesItemNumbers)
+        : null;
+      const nameColumn =
+        defineNameColumn<EventT>(
+          l('Event'),
+          order,
+          sortable,
+          false, // to use EntityLink without dates (separate column for that)
```

Comment:
It seems to fix a regression, might be worth to notify users about that through a ticket.
(Event search results have been changed the same way in MBS-9765.)

Reply:
Ok, created MBS-10583